easypy/misc.py

```diff
@@ -35,3 +35,22 @@ def get_all_subclasses(cls):
                                    for subclass in cls.__subclasses__()
                                    for rec_subclass in get_all_subclasses(subclass)]
     return [subclass for subclass in _all if not hasattr(subclass, "_%s__is_mixin" % subclass.__name__)]
+
+
+def is_context_manager(func):
+    from contextlib import contextmanager
+
+    _contextmanager_code_object = contextmanager(None).__code__
```

Comment:
Name is misleading. is_context_manager this way checks only, that it is contextmanager crated with contextlib wrapper. This does not have to be true